### ISSUE TITLE
Title bar becomes transparent when focus switches to second monitor (Ubuntu)

### ISSUE THREAD

**Issue by dushmanta05** [zed-industries/zed#16699]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

I'm experiencing an issue with the Zed editor on my Ubuntu device when using dual monitors. The problem occurs as follows:

- I have **Zed** editor open on my primary monitor (my laptop screen).
- When my pointer is focused on the Zed editor, everything displays correctly.
- However, when I switch focus to the second monitor, the Zed editor's title bar on the primary monitor becomes transparent.
- This transparency causes the title bar to show the other window(s) behind it on the primary monitor.
- The issue resolves when I refocus on the Zed editor on the primary monitor.

Steps to reproduce:

- Open Zed editor on the primary monitor.
- Open another application or window on the same monitor.
- Switch focus to the second monitor.
- Observe the Zed editor's title bar on the primary monitor.

### Environment

System:
  Host: dushmanta Kernel: 6.8.0-41-generic arch: x86_64 bits: 64 compiler: gcc
    v: 13.2.0
  Desktop: GNOME v: 46.0 Distro: Ubuntu 24.04 LTS (Noble Numbat)
  Display: Wayland

Zed:
  Version: 0.149.03

### If applicable, add mockups / screenshots to help explain present your vision of the feature

Here's how Zed looks on the primary monitor, as mentioned above

![Primary monitor](https://github.com/user-attachments/assets/7ee83a00-f315-43da-84e1-7728fdcd0e9b)

This is how Zed looks when focus is on the 2nd monitor

![Second monitor](https://github.com/user-attachments/assets/7c7ad635-99fa-4cb4-a86e-c8a7796e2eec)

### If applicable, attach your Zed.log file to this issue.

<details><summary>Zed.log</summary><pre>

</pre></details>

**Comment by tassianoalencar** [comment]
In Fedora 40 too!

**Comment by apricotbucket28** [comment]
This could be a theme-related issue. What theme are you using? 🙂

**Comment by dushmanta05** [comment]
@apricotbucket28 By theme, are you asking x11 or wayland? or is it something else? I've got pure Ubuntu with Wayland and no other theme.

**Comment by apricotbucket28** [comment]
> @apricotbucket28 By theme, are you asking x11 or wayland? or is it something else? I've got pure Ubuntu with Wayland and no other theme.

By theme I mean the Zed theme.

**Comment by dushmanta05** [comment]
@apricotbucket28 Ah sorry, I misunderstood, I'm using GitHub Dark Default theme currently

**Comment by apricotbucket28** [comment]
Thanks for the reply! I looked at the theme and it seems like it doesn't set the `title_bar.inactive_background` color key. 

However, this should have been fixed from even happening in https://github.com/zed-industries/zed/pull/15709. But it seems there are two more fallback places where the default color has to be changed from a transparent value to non-transparent: https://github.com/zed-industries/zed/blob/df2ffaaf77df339bfefc0078d100d7140340a1ba/crates/theme/src/default_colors.rs#L53 and https://github.com/zed-industries/zed/blob/df2ffaaf77df339bfefc0078d100d7140340a1ba/crates/theme/src/default_colors.rs#L151

So, basically:
1. You should create an issue in the [theme's repo ](https://github.com/mordfustang21/zed-github-dark) kindly asking for `title_bar.inactive_background` to be set.
2. Zed should use a non-transparent fallback color when the key is missing; I'll see if I can fix this later.
